src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnectionPoolKey.cs

```diff
@@ -74,6 +85,13 @@ private void CalculateHashCode()
                     _hashValue = _hashValue * 17 + _credential.GetHashCode();
                 }
             }
+            else if (_accessToken != null)
+            {
+                unchecked
```

Comment:
Why is the unchecked needed?  Is checked arithmetic enabled for this project?

Reply:
I can see unchecked being used in other parts of the SqlClient project.